### PR TITLE
add subsection about access methods of SPMP CSRs

### PR DESCRIPTION
**Pull request by ybc-alkaid**

### CODE REVIEW

File: spmp_spec.adoc [hunk 1]
```diff
@@ -202,4 +202,44 @@ Each bit of this register holds the on/off status of the corresponding SPMP entr
...
 An SPMP entry is activated only when both corresponding bits in spmpswitch and A field of spmpicfg are set. (i.e., spmpswitch[i] & spmp[i]cfg.A!=0)
 
-image::SPMP_domain_switch_register_format.png[title="SPMP domain switch register format (RV64)"]
+image::SPMP_domain_switch_register_format.svg[title="SPMP domain switch register format (RV64)"]
+
+
+
+
+=== Access Methods of SPMP CSRs
+How SPMP CSRs are accessed depends on whether the the `Sscsrind` extension is implemented or not.
```
Comment (Ddnirvana) [inline spmp_spec.adoc:211]: typo: double "the", please fix it @ybc-alkaid

**Review by Ddnirvana — COMMENTED** [review]
This change looks good to except the typo. Thanks.

File: spmp_spec.adoc (hunk 1 above)
Comment (ybc-alkaid) [reply spmp_spec.adoc]: done

**Review by Ddnirvana — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**